test/e2e/network_test.go

```diff
@@ -215,7 +215,7 @@ func retryGET(url string) (*http.Response, error) {
 func (tc *testContext) createService(name string, serviceType v1.ServiceType, selector metav1.LabelSelector) (*v1.Service, error) {
 	svcSpec := &v1.Service{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: name,
+			GenerateName: name + "-",
```

Comment:
Is it enough if we do it for the `Service`? I remember seeing this issue with [Jobs](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_windows-machine-config-operator/175/pull-ci-openshift-windows-machine-config-operator-master-azure-e2e-operator/1332051863398256640) too. So we might want to do this across the board for all objects we are creating in the e2e tests.

Reply:
Done !